ide/httpserver/src/org/netbeans/modules/httpserver/HttpServerSettings.java

```diff
@@ -428,54 +382,6 @@ Set<String> getGrantedAddressesSet() {
         return addr;
     }
 
-    Properties getMappedServlets() {
-        return mappedServlets;
-    }
-
-    /** Converts string into string that is usable in URL. 
-     *  This mangling changes some characters
-     */
-    static String mangle (String name) {
-        StringBuffer sb = new StringBuffer ();
```

Comment:
since this file is already touched why not changing this to StringBuilder?

Reply:
@lbownik red means removed